### ISSUE TITLE
Went Black after uploading data file, No LED, nothing on display

### ISSUE THREAD

**Issue by sarath341** [lewisxhe/Esp-badge#9]
After flashing the image file the screen went black. But the processor, serial monitor, wi-fi, everything works fine. Only the display went black & not even flashing while plugging it in as it does like before.

**Comment by frx08** [comment]
I solved updating to latest version of the epaper library [lewisxhe/GxEPD](https://github.com/lewisxhe/GxEPD) the master version not the release, i'm my case i was using 2.13 screen so i had also to update dependencies to #include <GxGDE0213B72B/GxGDE0213B72B.h>

**Comment by sarath341** [comment]
Yes, They have included a new header file, that is not updated in the example code it seems. Here is a detailed blog with full process https://www.tedizen.com/ttgo-tbadge-getting-started-tutorial/
